src/components/FuelSavingsForm.js

```diff
@@ -32,48 +32,48 @@ class FuelSavingsForm extends React.Component {
         <h2>Fuel Savings Analysis</h2>
         <table>
           <tbody>
-          <tr>
-            <td><label htmlFor="newMpg">New Vehicle MPG</label></td>
-            <td><FuelSavingsTextInput onChange={this.fuelSavingsKeypress} name="newMpg" value={fuelSavings.newMpg}/>
-            </td>
-          </tr>
-          <tr>
-            <td><label htmlFor="tradeMpg">Trade-in MPG</label></td>
-            <td><FuelSavingsTextInput onChange={this.fuelSavingsKeypress} name="tradeMpg" value={fuelSavings.tradeMpg}/>
-            </td>
-          </tr>
-          <tr>
-            <td><label htmlFor="newPpg">New Vehicle price per gallon</label></td>
-            <td><FuelSavingsTextInput onChange={this.fuelSavingsKeypress} name="newPpg" value={fuelSavings.newPpg}/>
-            </td>
-          </tr>
-          <tr>
-            <td><label htmlFor="tradePpg">Trade-in price per gallon</label></td>
-            <td><FuelSavingsTextInput onChange={this.fuelSavingsKeypress} name="tradePpg" value={fuelSavings.tradePpg}/>
-            </td>
-          </tr>
-          <tr>
-            <td><label htmlFor="milesDriven">Miles Driven</label></td>
-            <td>
-              <FuelSavingsTextInput
-                onChange={this.fuelSavingsKeypress}
-                name="milesDriven"
-                value={fuelSavings.milesDriven}/>
-              miles per
-              <select
-                name="milesDrivenTimeframe"
-                onChange={this.onTimeframeChange}
-                value={fuelSavings.milesDrivenTimeframe}>
-                <option value="week">Week</option>
-                <option value="month">Month</option>
-                <option value="year">Year</option>
-              </select>
-            </td>
-          </tr>
-          <tr>
-            <td><label>Date Modified</label></td>
-            <td>{fuelSavings.dateModified}</td>
-          </tr>
+            <tr>
```

Comment:
Once again, question about the spacing changes. Is this our eslint config doing this?

Reply:
@nickytonline This one looks like its from my text-editor.  I use an atom package `language-babel` to automatically calculate JSX spacing.  In this particular case, i think nudged it over accidentally.

Reply:
@TobiahRex - Would you be willing to rollback the whitespace changes in this PR to keep the diff minimal?

Reply:
@nickytonline & @coryhouse Please see note above ☝️

Reply:
@nickytonline & @coryhouse After checking again, i think this is a correct addition of whitespace.  The `<tr>` should appear as a **child** of `<tbody>` and not a **sibling** due to hierarchical precedence, correct?  If you'd sitll like me to revert back to what it was before, i can do that. please confirm.